src/transforms/index.js

```diff
@@ -1,30 +1,59 @@
-module.exports = {
-  arrayToList: require('../List/arrayToList'),
-  eitherToAsync: require('../Async/eitherToAsync'),
-  eitherToFirst: require('../First/eitherToFirst'),
-  eitherToLast: require('../Last/eitherToLast'),
-  eitherToMaybe: require('../Maybe/eitherToMaybe'),
-  eitherToResult: require('../Result/eitherToResult'),
-  firstToAsync: require('../Async/firstToAsync'),
-  firstToEither: require('../Either/firstToEither'),
-  firstToLast: require('../Last/firstToLast'),
-  firstToMaybe: require('../Maybe/firstToMaybe'),
-  firstToResult: require('../Result/firstToResult'),
-  lastToAsync: require('../Async/lastToAsync'),
-  lastToEither: require('../Either/lastToEither'),
-  lastToFirst: require('../First/lastToFirst'),
-  lastToMaybe: require('../Maybe/lastToMaybe'),
-  lastToResult: require('../Result/lastToResult'),
-  listToArray: require('../List/listToArray'),
-  maybeToAsync: require('../Async/maybeToAsync'),
-  maybeToEither: require('../Either/maybeToEither'),
-  maybeToFirst: require('../First/maybeToFirst'),
-  maybeToLast: require('../Last/maybeToLast'),
-  maybeToResult: require('../Result/maybeToResult'),
-  resultToAsync: require('../Async/resultToAsync'),
-  resultToEither: require('../Either/resultToEither'),
-  resultToFirst: require('../First/resultToFirst'),
-  resultToLast: require('../Last/resultToLast'),
-  resultToMaybe: require('../Maybe/resultToMaybe'),
-  writerToPair: require('../Pair/writerToPair')
+import arrayToList from '../List/arrayToList.js'
```

Comment:
would export { default as arrayToList } from '../List/arrayToList.js' be better here? that way you don't have to import the default object and then extract it, it'll just be a import { arrayToList } from 'crocks/transforms'

Reply:
If it works, I'm down. This does change the behaviour though, You'd have to `import * as crocks from 'crocks'` if you wanted to destructure. What do you think? @evilsoft